packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -120,16 +120,19 @@
     py-3.5;
 
   margin-inline-end: auto;
+  justify-content: center;
 
   .heading,
   .description {
     @apply block
-      break-words
-      p-0;
+    flex-none
+    break-words
+    p-0;
+    line-height: var(--calcite-font-line-height-relative-snug) !important;
```

Comment:
We should avoid using !important.

Reply:
I removed this and switched back to using `font-size: var(--calcite-font-size--1);`